docs/HTTP-batchsource.md

```diff
@@ -0,0 +1,206 @@
+# HTTP Batch Source
+
+Description
+-----------
+This plugin reads data from HTTP/HTTPS pages.
+Using paginated API to retrieve data is allowed.
+Data in JSON, XML, CSV, TSV, TEXT and BLOB formats is supported.
+
+Properties
+----------
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+### General
+
+**URL:** Url to fetch to the first page.
+The url must start with a protocol (e.g. http://).
+
+Supports {pagination.index} placeholder, neccessary for pagination
+type "Increment an index". For more information read pagination section.
+
+**HTTP Method:** Request HTTP method.
+
+**Headers:** Request HTTP headers for the.
+
+**Request body:** A body of the request.
+
+### Format
+
+**Format:** Format of pages. Used to determine how to convert pages into output records. Possible values are:<br>
```

Comment:
Format of the HTTP response. This determines how the response is converted into output records.

Reply:
done